logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -551,6 +555,42 @@ internal class MLSConversationDataSource(
             }
         }
 
+    override suspend fun getUserIdentity(userId: UserId) =
```

Comment:
Suggestion: we use this function almost every time we want to show the badge in a conversation's members list or in a 1:1 conversation, so can we get the conversation id from the AR when we calling this function?

Reply:
actually that function is used only in `OtherUserProfileScreen`. And there is no `conversationId` for the direct conversation with that user from the beginning.